DSPs/006-Onboarding-&-Engagement.md

```diff
@@ -0,0 +1,59 @@
+# Onboarding & Engagement Q1 2021
+
+**Author(s)**: @steffbrowne
+
+## Summary
+
+This proposal establishes a new workstream focused on improving the quality of builder engagement across the DAO. We’ll guide this work stream by
+- Facilitating effective & ongoing collaboration
+- Coordinating work bins to achieve measurable outcomes
+- Analyzing and reporting our progress to the DAO
+- Unifying work bins into learning milestones, so that we can evolve & iterate
+
+## Why should we fund this?
+
+We need to continue to have dedicated people maintaining and improving our processes as we continue to take on on a higher volume of projects and builders.
+
+## Anything else you'd like to add?
+
+This workstream covers the quality of engagement with and within the DAO at any of these stages:
+- Discovering the DAO
+- Onboarding
+- Project enrollment
+- Ongoing empowerment
+- Referrals
+
+Sample work bins will likely include these and more...
+- Mapping the [Member Journey](https://docs.google.com/document/d/1z47NnQ91T0OvlHFYXhwEY9SSEmz1Tj0MpNeT695XKJE/edit) (Steff)
+- Improve systems for organizing contact lists and running comms (Steff)
+- Keeping our Twitter active (Bernie)
+- Advising on procedures that improve quality of work (Ori)
+- Getting client feedback and measuring customer happiness (Rob)
+- Builder outreach campaign (Rob)
+
+This workstream will lead to new initiatives as well as build off of old / existing ones, such as,
+- [DSP 2: Q4 2020 Ops Roles](https://github.com/dOrgTech/Ops/blob/master/DSPs/002-Ops-Roles.md)
+- [DSP 4: Landing Page Redesign](https://github.com/dOrgTech/Ops/blob/master/DSPs/004-Landing-Page.md)
+- [DSP 5: Builder Referrals](https://github.com/dOrgTech/Ops/blob/master/DSPs/005-Builder-Referrals.md)
+- [Issue 70 Gamifying QA](https://github.com/dOrgTech/Ops/issues/72)
+- [Issue 72 Improve activation/onboarding funnel](https://github.com/dOrgTech/Ops/issues/70)
+
+Accountability:
+- Lead: Steff
+- Hour Commitments:
+  - Steff: 12 hrs/week
+  - Ori: 5-6 hrs/week
+  - Rob: 3-6 hrs/week
+  - Bernie: 1-2 hrs/week
+- Weekly Syncs: Tuesdays @ 10am EDT
+- [Task Board](https://github.com/orgs/dOrgTech/projects/9)
+- [Budget spreadsheet](TODO)
+
+## Cost x Timeframe
+
+$1,875/week ($22,500 total)
```

Comment:
Does dOrg have the necessary cash flow to sustain this kind of expense? Or are we dipping into our reserve substantially for this? It's difficult to tell from the [dOrg reserve spreadsheet](https://docs.google.com/spreadsheets/d/1mwYhzTNXSytzVtACZLu1V_EVTfjPKhGfHu-KhnBFESk/edit?usp=sharing).

Reply:
This may not be feasible in the scope of this DSP so I created an issue to address this evaluation: #85

Reply:
Here is my take on the cashflow question:

dOrg's revenue was ~$300k in 2019 and $1M in 2020, so we can *conservatively* project at least $2M in revenue for 2021.  Given a 10% dOrg Margin, this implies a $200k to fund DSPs.

The ops related DSPs for Q1 which are likely to recur are currently $22.5k for DSP 6 (Onboarding & Engagement) and ~$9k for DSP 7 (Bookkeeping), which totals $31.5k. Multiply by 4 quarters to get $126k, which is 63% of our hypothetical DSP funds available for the year.

This math also assumes that we keep spending on Onboarding & Engagement + Bookkeeping at the same level. But realistically, I think we would lower this engagement after Q1 since the heavy upfront investment in improving and automating many of our ops processes should hopefully payoff with a) less ops spending down the line and b) greater revenue from our improved efficiencies.

Reply:
Thank you, Ori! This is what I was looking for but could not come up with myself.